ci-operator/step-registry/ipi/conf/vsphere/ipi-conf-vsphere-ref.yaml

```diff
@@ -6,6 +6,20 @@ ref:
     requests:
       cpu: 10m
       memory: 100Mi
+  env:
+  - name: SIZE_VARIANT
+    default: ""
+    documentation: |-
+      The size of the cluster in one of our supported t-shirt values that is standard across all CI environments.
+
+      The sizes are:
+      * "" (default) - 4 vCPU, 16GB control plane nodes, default workers
+      * "compact" - 8 vCPU, 32GB control plane nodes, no workers
+      * "large" - 16 vCPU, 64GB+ control plane nodes, default workers, suitable for clusters up to 250 nodes
```

Comment:
This concerns me a little. vSphere env(s) has finite resources using large or xlarge probably won't be an option any time soon.

Reply:
I second that concern.  The expectation is that 'default/compact' are the only valid options for vSphere.  I'll lgtm it, but bear in mind the resource constraints in our CI environments.

Reply:
oh, and it still needs a rebase :)

Reply:
ok, the SIZE_VARIANT here we use in vsphere only set to "compact", because it's a common env, also used in other steps, "large" would not be used in vsphere. So maybe i can delete the note for "large"?

Reply:
yeah, let's do that.  thanks!

Reply:
@jcpowermac @rvanderp3 please help review again.